pkg/relay.go

```diff
@@ -150,50 +154,52 @@ func (rs *DefaultRelay) processValidator(ctx context.Context, payload []types.Si
 			return err
 		} else if !ok {
 			if rs.config.CheckKnownValidator {
-				return fmt.Errorf("not a validator")
+				return fmt.Errorf("%s not a known validator", registerRequest.Message.Pubkey.String())
 			} else {
-				logger.WithField("pubkey", pk.PublicKey).Debug("is not a known validator")
+				logger.
+					WithField("pubkey", pk.PublicKey).
```

Comment:
pubkey -> pub_key consistent w/ above changes

Reply:
But in all the places we are using `pubkey` instead of `pub_key`. Also, this way we are consistent with JSON fields that are written `pubkey` instead of `pub_key`

Reply:
sorry i misread the above change